django/forms/utils.py

```diff
@@ -114,10 +116,19 @@ def as_ul(self):
         if not self.data:
             return ''
 
+        li = format_html_join('', '<li>{}</li>', ((e,) for e in self))
+        if self.field:
+            return format_html(
+                '<div id="{}_{}"><ul class="{}">{}</ul></div>',
```

Comment:
Unfortunately as far as I can tell this doesn’t work in VoiceOver, due to the `ul` still being present. None of the content within the `ul` is announced. The content is correctly announced with the `div` alone, removing the list markup.

I also tried adding `role="presentation"` to the `ul` so screen readers bypass the list semantics, that worked as expected, but since the error list still is visually displayed as a list it’s pretty jarring to have to force it to not be announced as once.